src/flint/fq_nmod_rel_series.jl

```diff
@@ -567,6 +567,7 @@ function addeq!(a::fq_nmod_rel_series, b::fq_nmod_rel_series)
    ctx = base_ring(a)
    if a.val < b.val
       z = fq_nmod_rel_series(base_ring(a))
+      z.parent = a.parent # finalizer will error
```

Comment:
```suggestion
      z = parent(a)()
```
Does this also work or am I missing something?